jose-jwt/jose-jwt.csproj

```diff
@@ -93,7 +93,8 @@ RFC 7797:
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>3.0.0</VersionPrefix>
     <Authors>Dmitry Vsekhvalnov</Authors>
-    <TargetFrameworks>net40;net461;netstandard1.4;netstandard2.1</TargetFrameworks>
+    <!--<TargetFrameworks>net40;net461;netstandard1.4;netstandard2.1</TargetFrameworks>-->
+	<TargetFrameworks>net461;netstandard1.4;netstandard2.1</TargetFrameworks>
```

Comment:
- net40 should definitely go (imo)
- net461... if can delete that also would mean more clean-up possible